src/monopoly.js

```diff
@@ -134,6 +140,41 @@ export default class Monopoly {
     return this;
   }
 
+  loadPropertyArray(){
+    this.propertyArray = JSON.parse(fs.readFileSync('properties.JSON'));
+    this.propertyArray.forEach((property) => {
+      switch (property.functionID) {
+        case 1:
```

Comment:
Yes, My plan was to replace the ID numbers with strings once I finished testing some of the function concepts themselves.  Will look into the alternative solution now